portfolio/src/main/webapp/index.html

```diff
@@ -95,10 +95,10 @@ <h1>:D</h1>
       <!-- TODO(samsog@google.com):Merge Gallery Section -->
       <div class="section" id="gallery">
         <div class="gallery-wrapper">
-          <img src="" alt="selfie of me in a suit" class="gallery-card tall">
-          <img src="" alt="selfie of me and my friends" class="gallery-card">
-          <img src="" alt="selfie of me in a park" class="gallery-card">
-          <img src="" alt="selfie of me smiling" class="gallery-card wide">
+          <div style="background-image:url(/images/IMG_1865.JPG)" class="gallery-card tall"></div>
```

Comment:
[don't use `style` to define CSS stuff in HTML](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns).
Consider setting appropriate `id`-s to the different `<div>`-s, or adding a named class for each different background, and add it to the class lists appropriately.

Reply:
Moved to classes with this commit c650f14